src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: $card-main-color;
```

Comment:
Ensure that the variable `$card-main-color` is defined in your SCSS files. If it's not defined, this will cause a compilation error.